projects/gateway2/translator/sslutils/ssl_utils.go

```diff
@@ -59,3 +78,80 @@ func cleanedSslKeyPair(certChain, privateKey, rootCa string) (cleanedChain strin
 
 	return cleanedChain, err
 }
+
+type SslExtensionOptionFunc = func(ctx context.Context, in string, out *ssl.SslConfig) error
+
+func ApplyCipherSuites(ctx context.Context, in string, out *ssl.SslConfig) error {
+	cipherSuites := strings.Split(in, ",")
+	out.Parameters.CipherSuites = cipherSuites
+	return nil
+}
+
+func ApplyMinimumTlsVersion(ctx context.Context, in string, out *ssl.SslConfig) error {
+	if parsed, ok := ssl.SslParameters_ProtocolVersion_value[in]; ok {
+		out.Parameters.MinimumProtocolVersion = ssl.SslParameters_ProtocolVersion(parsed)
+		if out.Parameters.MaximumProtocolVersion != ssl.SslParameters_TLS_AUTO && (out.Parameters.MaximumProtocolVersion < out.Parameters.MinimumProtocolVersion) {
+			return eris.Errorf("maximum tls version %s is less than minimum tls version %s", out.Parameters.MaximumProtocolVersion.String(), in)
+		}
+	} else {
+		return eris.Errorf("invalid minimum tls version: %s", in)
+	}
+	return nil
+}
+
+func ApplyMaximumTlsVersion(ctx context.Context, in string, out *ssl.SslConfig) error {
+	if parsed, ok := ssl.SslParameters_ProtocolVersion_value[in]; ok {
+		out.Parameters.MaximumProtocolVersion = ssl.SslParameters_ProtocolVersion(parsed)
+		if out.Parameters.MaximumProtocolVersion != ssl.SslParameters_TLS_AUTO && !(out.Parameters.MinimumProtocolVersion <= out.Parameters.MaximumProtocolVersion) {
```

Comment:
Same here please 🙂 🙏 
```suggestion
		if out.Parameters.MaximumProtocolVersion != ssl.SslParameters_TLS_AUTO && (out.Parameters.MaximumProtocolVersion < out.Parameters.MinimumProtocolVersion) {
```